### PR TITLE
Minor changes to beefmen traumas

### PR DESCRIPTION
**Pull request by FernandoJ8**
## About The Pull Request

Replaces hypnotic stupor with monophobia and icky-ocky phobia with a phobia of conspiracies.
Makes the defines used for the preferences text clearer.

## Why It's Good For The Game

Hypnotic stupor's a soft license to grief/ not do your job which isn't great. I've replaced it with monophobia.

Icky ocky phobia becomes, in itself, immersion breaking, so I replaced it with a fear of conspiracies.

### CODE REVIEW

**Comment by Pepsilawn** [comment]
I dont think you need a lore reason when ocky icky phobia is also on the list

**Comment by SoreYew** [comment]
Or dont remove icky ocky and instead add paranoid of conspiracy as a choice

**Comment by SoreYew** [comment]
> > Or dont remove icky ocky and instead add paranoid of conspiracy as a choice
> 
> 
> 
> Ocky icky cant be obtained ic and was made for adminmemery why would it be a roundstart preference

Beefboy gets mad at ock ick is unique and fun, didnt know it was impossible IC though that kinda makes it different.

File: fulp_modules/_defines/species_defines.dm [hunk 1]
```diff
@@ -34,7 +34,7 @@
...
  * Beefman trauma pref defines
  */
-#define BEEF_TRAUMA_STRANGERS "Strangers"
-#define BEEF_TRAUMA_HALLUCINATIONS "Hallucinations"
-#define BEEF_TRAUMA_OCKY_ICKY "Ocky Icky"
-#define BEEF_TRAUMA_HYPNOTIC_STUPOR "Hypnotic Stupor"
+#define BEEF_TRAUMA_STRANGERS "Phobia of Strangers"
+#define BEEF_TRAUMA_HALLUCINATIONS "Severe Hallucinations"
+#define BEEF_TRAUMA_OCKY_ICKY "Phobia of Conspiracies"
+#define BEEF_TRAUMA_PACIFISM "Pacifism"
```
Comment (SgtHunk) [inline fulp_modules/_defines/species_defines.dm:40]: 1. Rename the define
2. Do we really want pacifism as a trauma? Surely there's some other trauma that can serve as a replacement that doesn't make a quirk useless.

**Review by SgtHunk — CHANGES_REQUESTED** [review]

File: fulp_modules/_defines/species_defines.dm (hunk 1 above)
Comment (FernandoJ8) [reply fulp_modules/_defines/species_defines.dm]: Changed it to monophobia and renamed the defines

**Comment by SgtHunk** [comment]
@FernandoJ8 hey man this is still a draft

**Pull request closed without merging**